daemon/daemon.go

```diff
@@ -266,10 +266,16 @@ func Start(sockFile string, c *cli.Context) error {
 	config := &daemonConfig{
 		Root: root,
 	}
-	exists, err := util.ObjectExists(config)
-	if err != nil {
-		return err
+
+	ignoreCfgFile := c.Bool("ignore-config-file")
```

Comment:
Agree with your concern. It's not a thorough solution for the problem. I will keep thinking it. And please share your idea if you have any later on.